syntax-markdown-commonmark12/src/test/resources/markdown12/specific/html2.test

```diff
@@ -3,7 +3,7 @@
 .inputexpect|markdown/1.2
 .# Test inline XHTML
 .#-----------------------------------------------------
-hello **<del>world</del>** <sup>beta</sup>
+hello **<del>world</del>**<br/><sup>beta</sup>
```

Comment:
I don't understand this change. In the input we wanted a space, not a line break.

Reply:
I just wanted to have a self-closing tag in at least one test. I could also move it to a new test, however.